descartes_moveit/src/ikfast_moveit_state_adapter.cpp

```diff
@@ -126,7 +125,7 @@ bool descartes_moveit::IkFastMoveitStateAdapter::getFK(const std::vector<double>
 
 void descartes_moveit::IkFastMoveitStateAdapter::setState(const moveit::core::RobotState& state)
 {
-  assert(static_cast<bool>(robot_state_));
+  ROS_ASSERT_MSG(static_cast<bool>(robot_state_), "'robot_state_' member pointer is null. Have you called initialize()?");
   *robot_state_ = state;
```

Comment:
I want to force the user to call `initialize()` before they touch this function. If we made only that change, then the `robot_state_` is going to get blown away when they do call `initialize()`. 

We'd need to change that function to check if the state variable has been initialized ahead of the rest of the system. If you want to do that we can; I just saw this as the most expedient solution.